docs/source/guides/quickstart.md

```diff
@@ -1,48 +1,149 @@
 # NuCypher Quickstart
 
-## Ursula
 
-### Install NuCypher
+## A Note about Side-Channels
 
+The NuCypher network does not store or handle an application's data; instead - it manages access *to* application data
+Management of encrypted secrets and public keys tends to be highly domain-specific. In other words, the surrounding architecture will vary greatly depending on the throughput, sensitivity and sharing cadence of the data in question, to name but a few influencing factors.
+In all cases however, NuCypher must be integrated with a storage and transport layer in order to function properly.
+Along with the transport of ciphertexts, the application also needs to include 
+facilities for several other pieces of cryptographic material, specifically, channels
+for Alice and Bob to discover each other's public keys, as well as a way to provide the policy's public key 
+to Bob and Enrico.
+ 
+##### Side Channel Data:
+  - Secrets:
+    - Message Kits (Ciphertext)
+    - Labels
+  - Identities:
+    - Alice Verifying Key
+    - Bob Encrypting Key
+    - Bob Verifying Key
+  - Policies:
+    - Policy Encrypting Key 
+
+
+## Connecting to the NuCypher Network
+
+### Running an Ethereum Node
+
+Operation of NuCypher cryptological characters [`Alice`, `Bob`, `Ursula`] requires
+a running Ethereum node and wallet (we recommend Geth).
+
+To run a Goerli connected Geth node:
 ```bash
-$ pip3 install -U nucypher
+$ geth --goerl
 ```
 
-### Run a Federated-Only Development Ursula
+### Provider URI
 
-```bash
-$ nucypher ursula run --dev --federated-only
+Nucypher uses the node's IPC-File to communicate, specified by `provider_uri`.
+By default in ubuntu, the path is `~/.ethereum/goerli/geth.ipc` - This path
+will also be logged to the geth-running console on startup. 
+
+### Ursula: Untrusted Re-Encryption Proxies
+
+When initializing an `Alice`, `Bob`, or `Ursula`, an initial "Stranger-`Ursula`" is needed to perform 
+the role of a `Teacher`, or "seednode".  The `known_nodes` will inform your character of all of the nodes
+they know about network-wide, then kick-off the automated node-discovery loop.
+
+Stranger `Ursula`s can be created by invoking the `from_seed_and_stake_info` method, then a `list` of `known_nodes`
+can be passed into any `Character`'s init:
+```python
+from nucypher.characters.lawful import Ursula, Alice
+
+ursula = Ursula.from_seed_and_stake_info(seed_uri=<URI>)  # ie. https://0.0.0.0:9151
+another_ursula = Ursula.from_seed_and_stake_info(seed_uri=<URI>)
+
+alice = Alice(checksum_address='0xdeadbeef',
```

Comment:
Ok, here's how initializing Alice actually works:
```
from nucypher.blockchain.eth.registry import InMemoryContractRegistry
from nucypher.blockchain.eth.interfaces import BlockchainInterfaceFactory

registry = InMemoryContractRegistry.from_latest_publication('goerli')
BlockchainInterfaceFactory.initialize_interface(provider_uri='file:///home/michwill/.ethereum/goerli/geth.ipc')

alice = Alice(checksum_address='0x287A817426DD1AE78ea23e9918e2273b6733a43D', known_nodes=[ursula], registry=registry)
```

Reply:
Ok, need a reactor. Now Alice learns

Reply:
And another way to produce (persistent) Alice or any other character:
```python
from nucypher.config.characters import AliceConfiguration
alice_config = AliceConfiguration(config_root='/home/michwill/tmp/alice', known_nodes={ursula}, start_learning_now=False, federated_only=False, checksum_address='0x287A817426DD1AE78ea23e9918e2273b6733a43D', provider_uri='file:///home/michwill/.ethereum/goerli/geth.ipc')
alice_config.initialize(password='test' * 4)
alice_config.keyring.unlock(password='test' * 4)
alice = alice_config.produce()
alice.start_learning_loop()
...
```

Reply:
We can mention that if you want to produce a persistent character, you need to create it from configuration, and have the relevant guide up. I think, that's what you wanted to do?